### DISCUSSION TITLE
Connection could not be verified. - qBittorrent v4.5.0 Web UI (64-bit)

### DISCUSSION THREAD

**Discussion by TheFatPanda-Dev** [jesec/flood#628] [General]
I run Flood on the same docker network (Synology server) as qBittorrent but for some reason, I can't connect to it.
I've tried all localhost IPs but still nothing.

Any ideas?

![not working](https://user-images.githubusercontent.com/46047869/215156582-05684e4b-43eb-4999-bb04-4cf7d9a86263.png)

**Comment by scottcandy34** [comment]
I Just ran into the same issue.
